IntegrationTests/common/script/Makefile.mk

```
@@ -42,8 +42,11 @@ MemPrints:
 	 elif [[ $${BASE_DIR} == "CombinedBarrelConfig" ]]; \
 	 then \
 	   ln -sf $(EMDATA)/MemPrintsCMBarrel MemPrints; \
+	 elif [[ $${BASE_DIR} == "CombinedConfig_FPGA2" ]]; \
+	 then \
+	   ln -sf $(EMDATA)/MemPrintsSplit MemPrints; \
 	 else \
-	   ln -sf $(EMDATA)/MemPrintsCM MemPrints; \
+	   ln -sf $(EMDATA)/MemPrints MemPrintsCM; \
```

@aryd The arguments need to be reversed here. The target is `MemPrintsCM` and the link name is always `MemPrints`.
```suggestion
	   ln -sf $(EMDATA)/MemPrintsCM MemPrints; \
```

---

Fixed.